sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py

```diff
@@ -344,3 +342,5 @@ class SystemEventNames(str, Enum):
     ContainerRegistryArtifactEventName = 'Microsoft.AppConfiguration.KeyValueModified'
 
     ContainerRegistryEventName = 'Microsoft.ContainerRegistry.ChartPushed'
+
+    KeyVaultAccessPolicyChangedEventName = 'Microsoft.KeyVault.VaultAccessPolicyChanged'
```

Comment:
You haven't changed the code gen script that was putting these in the wrong place?

Reply:
ignore this PR - i was just using it to confirm API view diff 

this is the one i'm working on - 
https://github.com/Azure/azure-sdk-for-python/pull/23852/